src/app/pages/edit-card/edit-card.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { EditCardComponent } from './edit-card.component';
+
+describe('EditCardComponent', () => {
+  let component: EditCardComponent;
+  let fixture: ComponentFixture<EditCardComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ EditCardComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(EditCardComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
shouldn't need testing file for final project

Reply:
Angular has a cli command that'll make components for you using "ng generate component" and the testing file comes with it.